### PR TITLE
refactor(server): Cleanup rabbitmq references and deprecate rabbitmq-related code

### PR DESCRIPTION
**Pull request by alexvy86**
## Description

Cleans up references to RabbitMq in configuration files and deployment instructions, and deprecates code that was related to RabbitMq but isn't used anymore and will be removed in a future major release.

## Reviewer Guidance

The review process is outlined on [this wiki page](https://github.com/microsoft/FluidFramework/wiki/PR-Guidelines#guidelines).

@hedasilv @znewton @pradeepvairamani particularly interested in any feedback related to the deprecations. Based on usage in this repo, they seem correct, but you'd know better if any of that code was repurposed or is still used somehow in other contexts even if the base r11s implementation doesn't seem to use it. Based on what I understood, the whole flow of "task sending/receiving".

### CODE REVIEW

File: server/routerlicious/packages/services-core/src/emptyTaskMessageSender.ts [hunk 1]
```diff
@@ -5,6 +5,10 @@
 
 import { ITaskMessage, ITaskMessageSender } from "./taskMessages";
 
+/**
+ * @deprecated - This was functionality related to RabbitMq which is not used anymore,
```
Comment (tylerbutler) [inline server/routerlicious/packages/services-core/src/emptyTaskMessageSender.ts:9]:
```suggestion
 * @deprecated This was functionality related to RabbitMq which is not used anymore,
```

No dashes for deprecations I think. @Josmithr always has to remind me about this. Maybe it's finally sinking in! 😂

Comment (alexvy86) [reply server/routerlicious/packages/services-core/src/emptyTaskMessageSender.ts:9]: Ah, yes, only params... copilot lied to me 😢

File: server/tinylicious/src/services/taskMessageSender.ts [hunk 2]
```diff
@@ -5,6 +5,11 @@
 
 import { ITaskMessage, ITaskMessageSender } from "@fluidframework/server-services-core";
 
+/**
+ * @deprecated This was functionality related to RabbitMq which is not used anymore,
+ * and will be removed in a future release.
+ */
+
```
Comment (Josmithr) [inline server/tinylicious/src/services/taskMessageSender.ts:12]: Nit
```suggestion
```

**Review by WayneFerrao — APPROVED** [review]

**Comment by alexvy86** [comment]
@znewton @hedasilv @pradeepvairamani friendly ping. Not blocking anything, just would be nice to keep the cleanup going :)

**Comment by alexvy86** [comment]
@hedasilv @znewton @pradeepvairamani since this has no functional changes, only deprecations, I'll merge this to keep the cleanup moving forward. Would still like to know if you have any thoughts on the things being deprecated; if they're really not used anywhere anymore.

**Pull request merged**